x-pack/plugin/security/qa/basic-enable-security/src/test/java/org/elasticsearch/xpack/security/EnableSecurityOnBasicLicenseIT.java

```diff
@@ -126,7 +126,7 @@ private void checkAuthentication() throws IOException {
         final Map<String, Object> auth = getAsMap("/_security/_authenticate");
         // From file realm, configured in build.gradle
         assertThat(ObjectPath.evaluate(auth, "username"), equalTo("security_test_user"));
-        assertThat(ObjectPath.evaluate(auth, "roles"), contains("security_test_role"));
+        assertThat(ObjectPath.evaluate(auth, "roles"), contains("security_test_role", "anonymous"));
```

Comment:
Do we guarantee order ? if not maybe `containsInAnyOrder()` is better here ?

Reply:
[Updated](https://github.com/elastic/elasticsearch/pull/53453/files#diff-25d7072ca2b09b64f3607d1a6af3666dR126). The order is preserved. But I don't think we need guarantee it.